rke/k8s_rke_system_images.go

```diff
@@ -6218,6 +6218,49 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
 		},
+		"v1.20.14-rancher1-2": {
+			Etcd:                      "rancher/mirrored-coreos-etcd:v3.4.15-rancher1",
+			Kubernetes:                "rancher/hyperkube:v1.20.14-rancher1",
+			Alpine:                    "rancher/rke-tools:v0.1.78",
+			NginxProxy:                "rancher/rke-tools:v0.1.78",
+			CertDownloader:            "rancher/rke-tools:v0.1.78",
+			KubernetesServicesSidecar: "rancher/rke-tools:v0.1.78",
+			KubeDNS:                   "rancher/mirrored-k8s-dns-kube-dns:1.15.10",
+			DNSmasq:                   "rancher/mirrored-k8s-dns-dnsmasq-nanny:1.15.10",
+			KubeDNSSidecar:            "rancher/mirrored-k8s-dns-sidecar:1.15.10",
+			KubeDNSAutoscaler:         "rancher/mirrored-cluster-proportional-autoscaler:1.8.1",
+			Flannel:                   "rancher/mirrored-coreos-flannel:v0.15.1",
+			FlannelCNI:                "rancher/flannel-cni:v0.3.0-rancher6",
+			CalicoNode:                "rancher/mirrored-calico-node:v3.17.2",
+			CalicoCNI:                 "rancher/mirrored-calico-cni:v3.17.2",
+			CalicoControllers:         "rancher/mirrored-calico-kube-controllers:v3.17.2",
+			CalicoCtl:                 "rancher/mirrored-calico-ctl:v3.17.2",
+			CalicoFlexVol:             "rancher/mirrored-calico-pod2daemon-flexvol:v3.17.2",
+			CanalNode:                 "rancher/mirrored-calico-node:v3.17.2",
+			CanalCNI:                  "rancher/mirrored-calico-cni:v3.17.2",
+			CanalControllers:          "rancher/mirrored-calico-kube-controllers:v3.17.2",
+			CanalFlannel:              "rancher/mirrored-coreos-flannel:v0.15.1",
+			CanalFlexVol:              "rancher/mirrored-calico-pod2daemon-flexvol:v3.17.2",
+			WeaveNode:                 "weaveworks/weave-kube:2.8.1",
+			WeaveCNI:                  "weaveworks/weave-npc:2.8.1",
+			AciCniDeployContainer:     "noiro/cnideploy:5.1.1.0.1ae238a",
+			AciHostContainer:          "noiro/aci-containers-host:5.1.1.0.1ae238a",
+			AciOpflexContainer:        "noiro/opflex:5.1.1.0.1ae238a",
+			AciMcastContainer:         "noiro/opflex:5.1.1.0.1ae238a",
+			AciOpenvSwitchContainer:   "noiro/openvswitch:5.1.1.0.1ae238a",
+			AciControllerContainer:    "noiro/aci-containers-controller:5.1.1.0.1ae238a",
+			AciGbpServerContainer:     "noiro/gbp-server:5.1.1.0.1ae238a",
+			AciOpflexServerContainer:  "noiro/opflex-server:5.1.1.0.1ae238a",
+			PodInfraContainer:         "rancher/mirrored-pause:3.2",
+			Ingress:                   "rancher/nginx-ingress-controller:nginx-0.49.3-rancher1",
+			IngressBackend:            "rancher/mirrored-nginx-ingress-controller-defaultbackend:1.5-rancher1",
+			IngressWebhook:            "rancher/mirrored-ingress-nginx-kube-webhook-certgen:v1.1.1",
+			MetricsServer:             "rancher/mirrored-metrics-server:v0.5.0",
```

Comment:
template.go needs to be updated to use the metrics server v0.5.0 add-on template